shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -127,6 +127,12 @@ @implementation FlutterViewController {
   fml::scoped_nsobject<UIScrollView> _scrollView;
   fml::scoped_nsobject<UIView> _keyboardAnimationView;
   MouseState _mouseState;
+  // Timestamp after which a scroll inertia cancel event should be inferred.
+  NSTimeInterval _scrollInertiaEventStartline;
+  // When an iOS app is running in emulation on an Apple Silicon Mac, direct trackpad input is
```

Comment:
what is "direct" trackpad input? in apple's terminology, "direct" input means input on the screen, so trackpad should be "indirect".

Reply:
Direct = direct connection from trackpad to UIKit. When `runningOnMac`, it goes through AppKit first. I can reword it to avoid confusion.

Reply:
yes please remove it since `.direct` is an input type that apple already uses.